client/client-multi/client-c/src/kaa_status.c

```diff
@@ -13,7 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+#include <stddef.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include "platform/stdio.h"
+#include "platform/sha.h"
```

Comment:
I'm absolutely sure, I've checked every C file ob both linux and econais platforms.